Edge/cie 3605 update telnet deprecation (#48)
SoftwareAG/c8y-releasenotes: content/release-10-15-0/edge-10-15-0-bundle/10_15_0.md

---
weight: 60
title: Release 10.15.0
layout: redirect
---

Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.

> Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.

Reviewer: ```suggestion
Download the Cumulocity IoT Edge version 10.15 from the [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.
```

### Removal of Telnet support

> Starting with {{< product-c8y-iot >}} Edge Release 10.15, accessing the Edge appliance remotely from the {{< product-c8y-iot >}} tenant using the Telnet protocol has been removed. {{< company-sag >}} recommends you to use the SSH protocol instead.

Reviewer: ```suggestion
Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the Cumulocity IoT tenant using the Telnet protocol has been removed. Software AG recommends you to use the SSH protocol instead.
```

Reviewer: We do not use variables in the release notes repo so far.

---
weight: 60
title: Release 10.15.0
layout: redirect
---

Cumulocity IoT Edge Release 10.15, includes the following improvements, fixes and known issues.

Download the Cumulocity IoT Edge version 10.15 from [Software AG Empower portal](https://empower.softwareag.com), based on the target hypervisor.

### Removal of Telnet support

> Starting Cumulocity IoT Edge Release 10.15, the support for Telnet protocol has been removed.

Reviewer: ```suggestion
Starting with Cumulocity IoT Edge Release 10.15, the support for the Telnet protocol has been removed.
```

> This change will affect you if you are trying to connect to the Edge appliance through remote access from the Cumulocity IoT tenant.

Reviewer 2: - [ ] Starting with Cumulocity IoT Edge Release 10.15, accessing the Edge appliance remotely from the {{< product-c8y-iot >}} tenant using Telnet protocol has been removed. {{< company-sag >}} recommends you to use the SSH protocol instead.

Author: Done

### Known issues

|<div style="width:100px">Ticket</div>|<div style="width:200px">Area</div>|Description
|:---|:---|:---

> For new enhancements in Cumulocity IoT, see [What's new](/release-10-11-0/whatsnew-10-11-0/).

Reviewer: ```suggestion
For new enhancements in Cumulocity IoT, see [What's new](/release-10-15-0/whatsnew-10-15-0/).
```
